scripts/remote/workers/run-perf.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+# Invariant: Docker image `pash` must exist.
+
+CONTAINER_NAME=pash-perf
+
+cleanup() {
+  docker container stop $CONTAINER_NAME
+  docker container rm $CONTAINER_NAME
+}
+
+run() {
+  docker exec $CONTAINER_NAME /bin/bash -c "cd /pash/evaluation/eurosys && $@"
+}
+
+cleanup
+docker run -itd --name pash-perf pash /bin/bash
+
+trap 'cleanup' EXIT
+trap 'echo "<<fail>>"' ERR
+
+if [ -d lock ]; then
+    echo "Busy on existing job."
+else
+    mkdir lock
+    trap 'rm -r lock' EXIT
+
+    docker exec $CONTAINER_NAME /bin/bash -c "cd /pash && git pull"
+    run ./execute_eurosys_one_liners.sh -m
+    run ./execute_unix_benchmarks.sh -m
```

Comment:
Does `execute_unix_benchmarks.sh` actually support `-m`? I think it just supports `-s` and `-l` :)